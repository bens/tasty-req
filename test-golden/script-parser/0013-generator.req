1: POST /
{ "a": {:g string},
  "b": {:g integer},
  "c": {:g double},
  "d": {:g boolean},
  "e": {:g maybe string},
}
---
{:r 1?}
