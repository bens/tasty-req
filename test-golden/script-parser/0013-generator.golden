[ Command
    { command'id = 1
    , command'always = False
    , command'method = "POST"
    , command'url = [ Right "/" ]
    , command'request_body =
        Just
          (M (C (InR
                   (ObjectF
                      (fromList
                         [ ( "a" , M (C (InL (Const ( "g" , ReqGen GenText )))) [] )
                         , ( "b" , M (C (InL (Const ( "g" , ReqGen GenInt )))) [] )
                         , ( "c" , M (C (InL (Const ( "g" , ReqGen GenDouble )))) [] )
                         , ( "d" , M (C (InL (Const ( "g" , ReqGen GenBool )))) [] )
                         , ( "e"
                           , M (C (InL (Const ( "g" , ReqGen (GenMaybe GenText) )))) []
                           )
                         ]))))
             [])
    , command'response_body =
        Just
          (M (C (InL
                   (Const ( "r" , RespRef (Ref 1 Request [] (fromList [])) ))))
             [])
    }
]
